src/renderer/routes/test_sequencer_panel/components/modals/CreatePlaceholderTestModal.tsx

```diff
@@ -0,0 +1,112 @@
+import { Button } from "@/renderer/components/ui/button";
+import { Dialog, DialogContent } from "@/renderer/components/ui/dialog";
+import { Input } from "@/renderer/components/ui/input";
+import {
+  createNewTest,
+  useDisplayedSequenceState,
+} from "@/renderer/hooks/useTestSequencerState";
+import { useState } from "react";
+import { toast } from "sonner";
+
+export const CreatePlaceholderTestModal = ({
+  isModalOpen,
+  setModalOpen,
+}: {
+  isModalOpen: boolean;
+  setModalOpen: (value: boolean) => void;
+}) => {
+  const [name, setName] = useState<string>("");
+  const [min, setMin] = useState<number>(0);
+  const [max, setMax] = useState<number>(0);
+  const [unit, setUnit] = useState("");
+
+  const { addNewElems } = useDisplayedSequenceState();
+
+  const handleCreate = async () => {
+    if (name === "") {
+      toast.error("Please enter a test name");
+    }
+    const res = await addNewElems([
+      createNewTest(
```

Comment:
maybe consider take the params as an object? This makes it a bit hard to understand what each parameter is for :(